src/vclock.rs

```diff
@@ -10,17 +10,21 @@
 //! assert!(a > b);
 //! ```
 
+// TODO: we have a mixture of language here with witness and actor. Clean this up
+
```

Comment:
Thinking about what might be less confusing for newcomers, I think `witness` might be more clear for them. What do you think about changing `Actor` to `Witness` to try to minimize confusion?

Reply:
I like `witness`, I'll make the change :)